franka_gazebo/models/stone/model.sdf

```diff
@@ -0,0 +1,104 @@
+<?xml version='1.0'?>
+<sdf version='1.7'>
+  <model name='stone'>
+    <link name='link'>
+      <pose>0.0 0.0 0.032 0 -0 0</pose>
```

Comment:
the 0.032 offset is causing the model spawn 3 cm above the table. Then it will fall down to the table without falling. The z axis should be set to 0